src/components/Text/Text.st.css

```diff
@@ -23,6 +23,7 @@
 
 .root {
     -st-states: typography(enum(smallTitle, runningText, listText, largeTitle)), mobile;
+    word-break: break-word;
```

Comment:
I'm not sure this is the behavior we want.
In addition, this CSS rule is deprecated: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

I'll check with our designers